scripts/build-circt.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
Please add an `set -eu`.

Reply:
Changed, but this has the minor drawback that the script now stops if it has been executed once, since the cloning of the CIRCT directory has already happened.